src/bus_monitor.cpp

```diff
@@ -10,6 +10,77 @@
 namespace panel
 {
 
+static void resetLEDState()
+{
+    std::cout << "Reset LED state" << std::endl;
+    std::vector<std::string> ledsPathOnBasePanel{
+        "/xyz/openbmc_project/led/groups/bmc_booted",
+        "/xyz/openbmc_project/led/groups/power_on",
+        "/xyz/openbmc_project/led/groups/enclosure_identify",
+        "/xyz/openbmc_project/led/groups/platform_system_attention_indicator",
+        "/xyz/openbmc_project/led/groups/partition_system_attention_indicator",
+        "/xyz/openbmc_project/led/groups/enclosure_fault"};
+
+    for (const auto& led : ledsPathOnBasePanel)
+    {
+        auto res = utils::readBusProperty<std::variant<bool>>(
+            "xyz.openbmc_project.LED.GroupManager", led,
+            "xyz.openbmc_project.Led.Group", "Asserted");
+
+        if (auto asserted = std::get_if<bool>(&res))
+        {
+            if (*asserted)
+            {
+                std::cout << "Led asserted = " << led << std::endl;
+                utils::writeBusProperty<bool>(
+                    "xyz.openbmc_project.LED.GroupManager", led,
+                    "xyz.openbmc_project.Led.Group", "Asserted", false);
+
+                utils::writeBusProperty<bool>(
+                    "xyz.openbmc_project.LED.GroupManager", led,
+                    "xyz.openbmc_project.Led.Group", "Asserted", true);
+            }
+        }
+        else
+        {
+            std::cerr
+                << "Failed to read asserted property. Unable to reset led "
+                << led << std::endl;
+        }
+    }
+}
+
+void PanelPresence::readEverBasePresentProperty(
```

Comment:
Please consider renaming this function (remove the "Ever" from the name).

Reply:
Done.